post_office/models.py

```diff
@@ -242,6 +242,7 @@ class Attachment(models.Model):
     name = models.CharField(_('Name'),max_length=255, help_text=_("The original filename"))
     emails = models.ManyToManyField(Email, related_name='attachments',
                                     verbose_name=_('Email addresses'))
+    mimetype = models.CharField(max_length=255, null=True)
```

Comment:
My thinking here is that django.core.messages.py::EmailMessage.attach is expecting None by default, so passing empty string if no value is explicitly set (instead of None) would do more harm than good.

BTW it looks like django 1.11 has some support for automatic mimetype detection if mimetype is None.

1.8 (1.9, 1.10 have no significant changes)
https://github.com/django/django/blob/1.8.18/django/core/mail/message.py#L305

1.11
https://github.com/django/django/blob/1.11.2/django/core/mail/message.py#L350